content/_data/changelogs/weekly.yml

```diff
@@ -1090,6 +1090,41 @@
     # pull: 3001 not notable enough
     # pull: 2995 not notable enough
     # pull: 2986 was reverted by 3013
+- version: "2.79"
+  date: 2017-09-17
+  changes:
+    - type: major bug
+      message: >
+        Fix random failures to use passphrase-protected ed25519 SSH private keys (regression in 2.73).
+      issue: 46754
+      pull: 3026
+    - type: major bug
+      message: >
+        Update remoting library from 3.11 to 3.12 to fix regression in Jenkins 2.68 when using non-writable home directories.
+      references:
+        - issue: 45755
+        - url: https://github.com/jenkinsci/remoting/blob/master/CHANGELOG.md#312
+          title: full changelog
+        - url: https://jenkins.io/doc/upgrade-guide/2.73/#known-issue-agent-connection-failures-involving-jenkins-masters-with-undefined-or-non-writable-home-directory
+          title: issue description in 2.73.1 upgrade guide
+      pull: 3025
+    - type: rfe
+      message: >
+        Add description of nodes to their remote API.
+      issue: 42854
+      pull: 2807
+    - type: bug
+      message: >
+        Disconnect node on ping timeout instead of leaving the channel half open.
+      pull: 3005
+      issue: 46680
+    - type: rfe
+      message: >
+        Internal: Require Java 8u101 to build Jenkins, as that's the minimum required to run it since 2.77.
+      pull: 3015
+    # pull 2998 (parent POM) doesn't seem notable enough
+    # pull 3000 impact unknown. Just some minor robustness fix?
```

Comment:
@daniel-beck yes, it is a robustness fix. Since the method is being used outside the core, it may deserve backporting. And it's better to add changelog in such case. I will create a follow-up PR.